### PR TITLE
Rebuildaarch64 and ppc64le addition1 arch

### PR DESCRIPTION
**Pull request by chrisburr**
See #1

### CODE REVIEW

**Comment by chrisburr** [comment]
@conda-forge-admin, please rerender

**Comment by chrisburr** [comment]
@conda-forge/libxkbcommon, this is ready for review/merge.

The travis-ci.com failure is expected and harmless. The real POWER build is on travis-ci.org and passes.

Once this is merged #1 should be closed.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -31,8 +31,6 @@ requirements:
     - {{ cdt('xorg-x11-proto-devel') }}
     - libxcb
     - xcb-proto
-    - xorg-xcb-proto
-    - xorg-libxcb
```
Comment (scopatz) [inline recipe/meta.yaml]: I think these are needed.

Comment (chrisburr) [reply recipe/meta.yaml]: These have been archived as they're duplicates of the un-prefixed (`xorg-`) packages. See conda-forge/xorg-xcb-proto-feedstock#2 and conda-forge/xorg-libxcb-feedstock#2.

Comment (chrisburr) [reply recipe/meta.yaml]: These have been archived as they're duplicates of the un-prefixed (xorg-) packages. See conda-forge/xorg-xcb-proto-feedstock#2 and conda-forge/xorg-libxcb-feedstock#2.

Comment (scopatz) [reply recipe/meta.yaml]: Oh I see. Thanks

**Pull request merged**